@W-21216935 | Documentation for AS2 v7.0.0 (#3143)
mulesoft/docs-connectors: as2/7.0/modules/ROOT/pages/as2-connector-examples.adoc

= AS2 Connector 7.0 Examples 

> Mule applications using AS2 Connector must include a global configuration to add external libraries specific to the active security mode. Because only one External Libs configuration is permitted per application, this element must be configured manually in the examples.

Reviewer: ```suggestion
Mule applications using Anypoint Connector for AS2 (AS2 Connector) must include a global configuration to add external libraries specific to the active security mode. Only one external libs configuration is permitted per application, so you must configure this element manually in the examples.
```

> These examples show how to configure send and receive messages in Anypoint Connector for AS2 (AS2 Connector):

Reviewer: ```suggestion
These examples show how to configure send and receive messages in AS2 Connector:
```

= AS2 Connector 7.0 Examples 

> The mule application that uses AS2 Connector needs a global configuration to add external libraries based on the security mode. Due to the restricition of only one External Libs global configuration allowed per mule application it needs to be configured manually in the examples below. Refer xref:as2-connector::as2-connector-upgrade-migrate.adoc[Configure External Libraries].

Reviewer 2: Mule applications using AS2 Connector must include a global configuration to add external libraries specific to the active security mode. Because only one External Libs configuration is permitted per application, this element must be configured manually in the examples.

> The following examples show how to configure send and receive messages in Anypoint Connector for AS2 (AS2 Connector):

Reviewer 2: This sentence should go at the top before the paragraph.
